### DISCUSSION TITLE
[Feature Request] View Option- Split View Icon

### DISCUSSION THREAD

**Discussion by ali-parvizi** [lxqt/lxqt#1906] [Ideas]
Hi everyone,

It would be great if we could have an icon for split view in the top toolbar or we could customize the icons in the top toolbar (maybe we already have this option, I couldn't find it), right now we can activate split view only through view menu.
I know that we have New Tab icon (which is pretty cool and handy) but with split view we can have different kind of controls over folders and files specially in big screens, 
In addition I would like to suggest another idea, having Options for Split View, something like splitting the view in three parts (I greatly appreciate such an option :)) 

Bests,

**Comment by stefonarch** [comment]
> right now we can activate split view only through view menu.

Well, not only: press F6 ;)

**Reply by ali-parvizi** [reply]
yeah, the point is that I changed my shortcut settings, so F6 is not working for this task, AND also I prefer to minimize using shortcut keys as much as possible, it gets confusing after some time :)

**Reply by stefonarch** [reply]
It was not meant to say this feature request isn't valid. As for me an additional  button could be ok.

**Reply by ali-parvizi** [reply]
I understand, 
I tried to explain why a button might be a better choice than a shortcut key in some cases.
